RecoVertex/AdaptiveVertexFinder/plugins/TemplatedVertexArbitrator.h

```diff
@@ -42,8 +42,10 @@
 
 //#define VTXDEBUG
 
-const unsigned int nTracks(const reco::Vertex &sv) { return sv.nTracks(); }
-const unsigned int nTracks(const reco::VertexCompositePtrCandidate &sv) { return sv.numberOfSourceCandidatePtrs(); }
+inline const unsigned int nTracks(const reco::Vertex &sv) { return sv.nTracks(); }
```

Comment:
I would embed this in place using constexpr if (std::is_same ... etc  
or at least add some sort of namespace....

Reply:
Is this really an header file?
it is a producer, so it shall be final and most probably is missing just two macros...

Reply:
still a function named nTracks shall not be allowed in the global name space so inline even if it becomes  a .src

Reply:
I guess we can just merge this header in to its source file ( https://github.com/cms-sw/cmssw/blob/master/RecoVertex/AdaptiveVertexFinder/plugins/VertexArbitrators.cc )

Reply:
@cms-sw/reconstruction-l2 , any objection on merging `TemplatedVertexArbitrator.h` in to `VertexArbitrators.cc` which is the only src file which includes it

Reply:
I think that it's fine to merge this to RecoVertex/AdaptiveVertexFinder/plugins/VertexArbitrators.cc

IIUC, the nTracks should be moved to the anonymous namespace (the easiest; else it's `is_same` with sfinae)

Reply:
`TemplatedVertexArbitrator.h` is now merged in to `VertexArbitrators.cc`